services/horizons/__init__.py

```diff
@@ -16,12 +16,13 @@
 from lucupy.minimodel import NonsiderealTarget, TargetTag, Site
 
 
+@final
 @dataclass
 class HorizonsAngle:
     """
     Angle in radians.
     """
-    microarcsecsPerDegree: float = 60 * 60 * 1000 * 1000
+    MicroarcsecsPerDegree: Final[float] = 60 * 60 * 1000 * 1000
```

Comment:
Even better `MICRO_ARCSECS_PER_DEGREE` I think is python guideline for variables like that. (globals at least)

Reply:
Makes sense, because we also have constants written in CAPS in lots of other places. I'll change it.